WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -40346,19 +40368,38 @@ e_latin_empire = {
 	capital = 496 # Byzantion
 	
 	short_name = yes
+	dynasty_title_names = no
 	
 	
 	allow = {
 		OR = {
+			has_romance_culture_trigger = yes
+			culture_group = latin
 			religion = catholic
 			is_heresy_of = catholic
 		}
-		e_byzantium = {
-			has_holder = no
+
+		NOT = {
+			culture = greek
+			culture = roman
+			religion = orthodox
+			religion = iconoclast
 		}
+
+		NOT = { has_landed_title = e_byzantium }
+
 		e_roman_empire = {
 			has_holder = no
 		}
+		
+		496 = { # Constantinople
+			owner = {
+				OR = {
+					is_liege_or_above = ROOT
+					character = ROOT
+				}
+		}
+
```

Comment:
In the file Latin Empire's circumstances are as follows:

1. You  must control the Latin Empire's historic lands (Thrake, Kaliopolis, and Constantinople)
2. You must not be Orthodox or Iconoclast (post-Schism)
3. You must not hold the Byzantine Empire
4. Rome has not been restored

These are near identical to the Vanilla rendition. Besides, the Latins have always wanted to hold the City of the World's Desire.